src/FSharp.Data.GraphQL.Shared/Introspection.fs

```diff
@@ -166,7 +166,7 @@ let rec __Type =
             | Some name ->
                 let found = findIntrospected ctx name
                 found.PossibleTypes |> Option.map Array.toSeq)
-        Define.Field("enumValues", Nullable (ListOf __EnumValue), description = null,
```

Comment:
This was causing the introspection schema to have `Some null` as description instead of `None`, so I created an overload that does not ask for a description instead.